plugins/aws_sts/app/views/samson_aws_sts/_fields.html.erb

```diff
@@ -0,0 +1,45 @@
+<fieldset>
+  <legend>AWS STS Tokens</legend>
+  <div class="col-lg-offset-2">
+    <div class="form-group">
+      <div class="col-lg-6">
+        <p>Tokens will be requested using the following user: <%= SamsonAwsSts::Client.new.caller_user_id %></p>
+        <span class="help-block">
+          Use the following env variables to change which user to authenticate with:
```

Comment:
"in the project settings"

Reply:
"(set secret key via secret:// and secret manager)"

Reply:
these are actually samson env vars, I'll update to make it clearer